modules/cpmso-ts-mhc-etcd-degraded.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-mhc-etcd-degraded_{context}"]
+= Recovering a degraded etcd Operator after a machine health check operation
+
+While performing remediation, the machine health check might delete a control plane machine that is hosting etcd. If the etcd member is not reachable at that time, the etcd Operator becomes degraded. Manual intervention is required to force the etcd Operator to remove the failed member and restore the cluster state.
+
+.Procedure
+
+. List the control plane machines in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines -l machine.openshift.io/cluster-api-machine-role==master -n openshift-machine-api
+----
++
+#How do they recognize which one is the problem?#
```

Comment:
needinfo here

Reply:
Looks like if you run `oc get machines -n openshift-machine-api -o wide`, the unhealthy machine might have `STATE` = `stopped`. Can I just add `-o wide` to the command above and tell them to look for that `STATE`?